_includes/title.html

```diff
@@ -19,10 +19,14 @@
     <h1>
         {{ page.title | xml_escape }}
         <small>{{ page.subtitle }}</small>
+        
         {% if edit_path %}
             <a class="btn btn-primary pull-right" href="{{ edit_path }}">Edit on Github</a>
         {% elsif show_news_link %}
             <a class="btn btn-primary pull-right" href="/news/atom.xml">Subscribe via Atom</a>
         {% endif %}
     </h1>
+    {% if page.author %}
```

Comment:
Hmmm, that makes sense -- thanks for the explanation! I'd like to look into ways to prominently display the contributor names in a way that says "this person contributed" but still keeps it communal and open. I have some ideas that I'll flesh out when I get the chance, but the changes I want to make are mainly just presentation of this info. Maybe a "special thanks" section or similar. The point on contactability is also an important one, so I might try to use the GitHub user cards.

Anyway, for now I think this is fine. Just make sure that you modify it so that this line only shows up in tutorials, not projects or other pages.